voluptuous/error.py

```diff
@@ -142,11 +142,11 @@ class BooleanInvalid(Invalid):
 
 
 class UrlInvalid(Invalid):
-    """The value is not a url."""
+    """The value is not an URL."""
```

Comment:
"a" is grammatically correct English for an initialism starting with a vowel.

Reply:
Will adjust